lib/src/firestore.dart

```diff
@@ -33,8 +33,7 @@ class Firestore extends JsObjectWrapper<firestore_interop.FirestoreJsImpl> {
       return null;
     }
 
-    return _expando[jsObject] ??= Firestore._fromJsObject(jsObject
-      ..settings(firestore_interop.Settings(timestampsInSnapshots: true)));
```

Comment:
Uh...don't we want this to stay here?

Reply:
That’s actually a breaking change now to call `timestampsInSnapshots: true`. The default as specified in the documentation is to have Date in snapshot. There was a bug in dartifyDate that prevented this to work properly that I fixed.

Initially I wanted to add another settings to allow using DateTime or Timestamp at the app level, but I could not manage to have dartify work differently in the 2 usages depending on the firestore app used and actually the existing setting works perfectly (we get js Date or js Timestamp so it easy to do the proper conversion in dartify).

It was used the wrong way: we were forcing to use Timestamp and converting every timestamp to Date loosing the microsecond precision, it seems it was just here to hide a warning that is actually relevant. If we decide to make  `timestapsInSnapshots: true` the default, that will break existing apps and I don’t want to be the one changing this (although I’m all in favor). The warning in the console is a good step to have developers migrating to Timestamp which might be the only option in the future.

As shown here (https://github.com/tekartik/firebase-dart/blob/ebc04f57317cbe380eff14f4c6d678f1b37c1724/test/firestore_test.dart#L976-L1015), when you use `timestampsInSnapshots: true` you only get timestamp. However you can write using either DateTime or Timestamp